ctapipe_io_lst/__init__.py

```diff
@@ -58,6 +60,18 @@ class TriggerBits(IntFlag):
     OTHER = CALIBRATION | SINGLE_PE | SOFTWARE | PEDESTAL | SLOW_CONTROL
 
 
+class PixelStatus(IntFlag):
+    RESERVED_0 = auto()
+    RESERVED_1 = auto()
+    HIGH_GAIN_STORED = auto()
+    LOW_GAIN_STORED = auto()
+    SATURATED = auto()
+    PIXEL_TRIGGER_1 = auto()
+    PIXEL_TRIGGER_2 = auto()
+    PIXEL_TRIGGER_3 = auto()
+
```

Comment:
Hi @maxnoe,
to what do the pixel_trigger_1,2,3 correspond? (perhaps a comment?)

Reply:
This is explained in the R1 document, https://forge.in2p3.fr/dmsf/files/8627, section A.5:

> bits 5-7" the last 3 bits are for recording pixel-wise (or sector-wise) trigger info (contents to be
determined)

I will add a link to the document.

Reply:
For current LST data, they seem to always be 0